src/components/variant-aspects-layout/aspect-window/aspect-window-base.tsx

```diff
@@ -77,12 +79,31 @@ export const AspectWindowBase = ({
         break
       case AspectDescriptorTypes.Table:
         content = (
-          <AspectTableView aspect={aspect} shouldAddShadow={shouldAddShadow} />
+          <AspectTableView
+            aspect={aspect}
+            shouldAddShadow={shouldAddShadow}
+            columnRef={columnRef}
+          />
         )
         break
     }
   }
 
+  useEffect(() => {
+    const margin = columnRef.current?.clientWidth || 0
+    const shadows =
+      contentRef.current?.parentElement?.firstElementChild?.children
+
+    if (!shadows) return
+
+    for (let i = 0; i < shadows.length; i++) {
+      const element = shadows[i]
+      if (element.id === 'leftShadow') {
+        ;(element as HTMLElement).style.left = `${margin}px`
```

Comment:
Looks like a typo (';' in the beginning of line)

Reply:
It is eslint requirement